### ISSUE TITLE
DL3013: argument mistaken for package

### ISSUE THREAD

**Issue by SanderDelden** [hadolint/hadolint#1064]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

When setting `--root-user-action ignore` in the pip install command `ignore` is interpreted as a Python package and Hadolint wants me to pin a version for it. `ignore` is a argument and should not be mistaken for a package.

```bash
Dockerfile:3 DL3013 warning: Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`
```

**Version**

```bash
Haskell Dockerfile Linter 2.12.0
```

### Steps to reproduce the behavior

**Dockerfile**

```Dockerfile
FROM python:3.12.8-alpine3.20

RUN pip install --no-cache-dir --root-user-action ignore poetry==1.8.5
```

**Additional environment details (OS, stack version, etc.)**

```bash
ProductName:		macOS
ProductVersion:		14.5
BuildVersion:		23F79
Darwin Kernel Version 23.5.0: Wed May  1 20:12:58 PDT 2024; root:xnu-10063.121.3~5/RELEASE_ARM64_T6000
```

**Comment by jamesquilty** [comment]
I see exactly the same issue with Hadolint 2.14.0. While this does appear to be a bug which should be fixed, there is a workaround: use an `=` in the option, thus `--root-user-action=ignore`. This appears to be acceptable to both pip and Hadolint.
